package.json

```diff
@@ -62,6 +62,7 @@
     "ember-infinity": "0.2.3",
     "ember-load-initializers": "0.5.1",
     "ember-lodash": "4.17.0",
+    "ember-markdown-it": "0.0.5",
```

Comment:
Two things here:

1) You don't seem to be using this [addon](https://github.com/HeroicEric/ember-markdown-it), except that I suspect it's helping install markdown-it for you, because the markdown-it line in bower.json is not sufficient to make markdown-it available to the app.

2) Just to clarify, we are not using this addon, and making a custom helper that uses markdown-it instead, because we need to override specific things, like disabling images?